src/engraving/dom/select.cpp

```diff
@@ -227,8 +227,8 @@ ChordRest* Selection::firstChordRestInRange(track_idx_t track) const
         return nullptr;
     }
 
-    for (track_idx_t track = staff2track(staffStart()); track < staff2track(staffEnd()); ++track) {
-        EngravingItem* e = firstCRSegment->element(track);
+    for (track_idx_t strack = staff2track(staffStart()); strack < staff2track(staffEnd()); ++strack) {
```

Comment:
let's instead rename the outer one to `preferredTrack`, as that's more meaningful
same below

Reply:
All done already?